bottlecap/src/logger.rs

```diff
@@ -4,7 +4,7 @@ use log::{LevelFilter, Log, Metadata, Record};
 pub struct SimpleLogger {}
 
 impl SimpleLogger {
-    pub fn init(level: LogLevel) -> Result<(), log::SetLoggerError> {
+    pub fn init(level: &LogLevel) -> Result<(), log::SetLoggerError> {
```

Comment:
LogLevel derivces Copy and is a small enum, so there isn't much point borrowing (with &) it here vs just copying/moving (without).

Reply:
The one we are using doesn't, so I'm going to add it, remove the borrowing and move it directly!